packages/express/ami/ami-rekey.sh

```diff
@@ -38,16 +38,16 @@ do
     sleep 5
 done
 
-until docker top $(docker ps | grep _openemr | cut -f 1 -d " ") | grep httpd &> /dev/null
+until docker top $(docker ps | grep -- -openemr | cut -f 1 -d " ") | grep httpd &> /dev/null
 do
     sleep 3
 done
 
 # reset password
-docker exec $(docker ps | grep _openemr | cut -f 1 -d " ") /root/unlock_admin.sh $(curl http://169.254.169.254/latest/meta-data/instance-id)
+docker exec $(docker ps | grep -- -openemr | cut -f 1 -d " ") /root/unlock_admin.sh $(curl http://169.254.169.254/latest/meta-data/instance-id)
```

Comment:
Just checking, are these all supposed to be running in docker compose? If so, instead of using `docker ps | grep` could we use `docker compose exec …`?